administrator-manual/en/disaster_recovery.rst

```diff
@@ -59,15 +59,18 @@ Please, follow below steps:
 5. Map network interface names from the backup to the running system.
    This step is required only if :guilabel:`Restore network configuration` option is enabled.
 
-6. Click the :guilabel:`Restore` to start the restore process.
+6. It's **mandatory** to keep the option :guilabel:`Reinstall packages` **enabled**!
+   Its deactivation could introduce unexpected behavior on restored system!
```

Comment:
```suggestion
6. During disaster recovery, to obtain a machine identical to original one, it's **mandatory** to keep the option :guilabel:`Reinstall packages` **enabled**!
   Its deactivation could introduce unexpected behavior on restored system!
```

Reply:
Ok for me

Reply:
We have to consider that disaster recovery is a "special" version of restore, both for config and data: you want to replicate of the scenario you had before.
If you want a partial restore it is not a complete disaster recovery.